.travis.yml

```diff
@@ -4,6 +4,9 @@ sudo: false
 
 node_js:
   - 0.10
+  - 0.12
+  - 4
+  - 6
 
```

Comment:
I would also add `   - node` to test on latest stable Node.js release.

Reference : https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Available-Versions